eZ/Publish/Core/Repository/ContentService.php

```diff
@@ -823,6 +824,14 @@ protected function buildSPILocationCreateStructs(array $locationCreateStructs)
                 );
             }
 
+            if (!isset(Location::SORT_FIELD_MAP[$locationCreateStruct->sortField])) {
+                $locationCreateStruct->sortField = Location::SORT_FIELD_NAME;
```

Comment:
Personally I would throw in this case, but that might be rather for kernel 8.x due to BC.

Reply:
yes, more of a 8.x way

Reply:
This is failsafe for when someone set it to not implemented sortField - which did nothing prev this PR - and after updating will create the new object of said ContentType, which now would break.